src/components/Todo/TodoContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import { Todo } from './Todo';
+import { TodoContainerProps } from './interfaces';
+
+export const TodoContainer: React.FC<TodoContainerProps> = ({
+  id,
+  title,
+  description,
+  deleteTodo,
+}) => {
+  const [avaliableOptions, setAvaliableOptions] = useState([
+    'Todo',
+    'In progress',
+    'Done',
+  ]);
```

Comment:
What does this state do?
Config eslint on your code editor. It should highlight this kind of errors

Reply:
removed state,
yes, i had a warning, thought warnings = okay, no problem, just warning)